dist/textAngular-sanitize.js

```diff
@@ -378,7 +378,7 @@ function htmlParser(html, handler) {
   parseEndTag();
 
   function parseStartTag(tag, tagName, rest, unary) {
-    tagName = angular.lowercase(tagName);
+    tagName = (tagName || "").toLowerCase();
```

Comment:
This is a breaking change. `angular.lowercase(null)` used to return `null` resulting in `tagName` staying null, with this chance if tagName is null, it will become `""` due to this change.

You probably want: `tagName = tagName ? tagName.toLowerCase() : tagName` to not make it a breaking change.

Note: this applies for all your changes.

Reply:
`tagName = tagName ? tagName.toLowerCase() : tagName` would be a breaking change too -- `tagName` might be, for instance, a number, so `tagName.toLowerCase()` would fail.

To make this non-breaking, would have to be: `angular.isString(tagName) ? tagName.toLowerCase() : tagname`

I can make that change now.